lib/components/chat_history/twitch/follow_event.dart

```diff
@@ -1,6 +1,8 @@
+// ignore_for_file: unused_import
```

Comment:
I think we dont need this one? Looks like all the imports are used.

Reply:
Good catch not sure where that came from